src/Microsoft.OpenApi.OData.Reader/OpenApiExtensions/OpenApiEnumValuesDescriptionExtension.cs

```diff
@@ -0,0 +1,71 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OpenApi.Interfaces;
+using Microsoft.OpenApi.OData.Common;
+using Microsoft.OpenApi.Writers;
+
+namespace Microsoft.OpenApi.OData.OpenApiExtensions;
+
+/// <summary>
+/// Extension element for OpenAPI to add enum values descriptions.
+/// Based of the AutoRest specification https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-enum
+/// </summary>
+internal class OpenApiEnumValuesDescriptionExtension : IOpenApiExtension
+{
+	/// <summary>
+    /// Name of the extension as used in the description.
+    /// </summary>
+	public string Name => "x-ms-enum";
```

Comment:
static or static readonly since it's constant?

Reply:
this property is required by the interface, and this syntax makes it readonly.